pages/common/gulp.md

```diff
@@ -0,0 +1,11 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
+
+- Run the default task defined inside gulpfile.js:
+
+`gulp`
+
+- Run specific task:
```

Comment:
We don't need to write a lot about gulp, just a friendly hint that can save users some time. How about changing the command description to "Run specific tasks from the gulpfile:"?